src/SDKs/ProvisioningServices/Management.ProvisioningServices/Microsoft.Azure.Management.ProvisioningServices.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ProvisioningServices</PackageId>
+    <Description>Provides management capabilities for Microsoft Azure IoT Device Provisioning Service.</Description>
+    <AssemblyTitle>Microsoft Azure Provisioning Service Management</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.ProvisioningServices</AssemblyName>
+    <VersionPrefix>0.9.0-preview</VersionPrefix>
+    <PackageTags>Microsoft Azure IoT Provisioning Services;IoT Device Provisioning Service Management;IoT Provisioning Service;</PackageTags>
+  </PropertyGroup>
```

Comment:
@montgomp please remove lines 10-17
Add packageReleaseNotes
Model your project looking at the other SDK project files. you are missing 1 imports at the bottom of your project file.

Reply:
added the import statement. Moved PackageReleaseNotes node into PropertyGroup that contains other package information

Reply:
Removing those lines will break compilation as it would remove a closing </PropertyGroup> and remove the TargetFrameworks node.